tests/api/test_api_flow.py

```diff
@@ -75,28 +75,3 @@ def test_simple_file_upload(client, location, full_input_record):
         'https://localhost:5000/records/{}/files'.format(recid)
     )
     assert response.json['links']['files'] == expected_files_url
-
-
-def test_new_api_integration(client):
-    """Test integration with new REST API.
-
-    TODO: Once integration is complete, this test can be removed since the
-          above will cover it.
-
-    NOTE: we are mocking implementation out bc we just want to test
-          integration.
-    """
-    expected_response_keys = set(['hits', 'links', 'aggregations'])
-    expected_metadata_keys = set([
-        'access_right', 'resource_type', 'creators', 'titles'
-    ])
-
-    # Get published bibliographic records
-    response = client.get('/rdm-records', headers=HEADERS)
-
-    assert response.status_code == 200
-    response_keys = set(response.json.keys())
-    assert response_keys == expected_response_keys
-    for r in response.json["hits"]["hits"]:
-        metadata_keys = set(r["metadata"])
-        assert metadata_keys == expected_metadata_keys
```

Comment:
That's ok. I will revisit the search tests when I am done with search in invenio-records-resources.

Reply:
🚀